Userland/Libraries/LibGUI/ComboBox.cpp

```diff
@@ -46,8 +47,10 @@ class ComboBoxEditor final : public TextEditor {
             if (is_focused())
                 set_focus(false);
             event.accept();
-        } else
+        } else{
```

Comment:
This isn't formatted properly

```suggestion
        } else {
```